product_docs/docs/pgd/5/routing/administering.mdx

```diff
@@ -38,16 +36,16 @@ SELECT bdr.routing_leadership_transfer('group1', 'node1');
 
 ### Using PGD CLI
 
-You can use the [`switchover`](/pgd/latest/cli/command_ref/pgd_switchover/) command to perform a switchover operation.
+You can use the [`group set-leader`](/pgd/latest/cli/command_ref/group/set-leader/) command to perform a switchover operation.
 
 For example, to switch the write leader from node `node1` to node `node2` in group `group1`, use the following command:
 
 ```sh
-pgd switchover --node-group group1 --node-name node1 --method strict --timeout 10s
+pgd group group1 set-leader node1 --method strict --timeout 10s
```

Comment:
The flag `--method` doesn't exist. Instead , boolean flags viz., `--fast` and `--strict` are added